BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -342,17 +342,19 @@ extension BlueprintView {
                         controller.update(node: child, appearanceTransitionsEnabled: true)
                     }
                 } else {
-                    let controller = NativeViewController(node: child)
-                    newChildren.append((path: path, node: controller))
-                    
+                    var controller: NativeViewController!
+
+                    // Building the view and applying the initial layout and update need to be wrapped in
+                    // performWithoutAnimation so they're not caught up inside an occuring transition.
                     UIView.performWithoutAnimation {
+                        controller = NativeViewController(node: child)
                         child.layoutAttributes.apply(to: controller.view)
+                        controller.update(node: child, appearanceTransitionsEnabled: false)
                     }
-                    
+
+                    newChildren.append((path: path, node: controller))
                     contentView.insertSubview(controller.view, at: index)
```

Comment:
Note the update is now happening _after_ the insert subview... Would this break anything? I guess not; _unless_ the update call walks the view hierarchy?

Reply:
And the only element I know of that walks the view hierarchy in SPOS (ok, Dashboard) goes down the tree: https://git.sqcorp.co/projects/IOS/repos/register/browse/Frameworks/SquareSalesReports/Sources/Elements/ExclusiveTouch.swift#29

Reply:
I thought about that a little, but I would hope if you're doing that you're not making assumptions about how blueprint works; and instead, using `didMoveToSuperview` or something. Do you think that's reasonable?

Reply:
Yeah, definitely – just figured I'd call it out

Reply:
I don't think there's any harm to inserting the view in the block though, maybe I should just do that